server/src/main/java/io/crate/planner/operators/LogicalPlanner.java

```diff
@@ -112,8 +114,8 @@
  * Planner which can create a {@link ExecutionPlan} using intermediate {@link LogicalPlan} nodes.
  */
 public class LogicalPlanner {
-
-    private final IterativeOptimizer optimizer;
+    @VisibleForTesting
```

Comment:
Apologies if i didn't spot it, but is this actually used in tests?

Reply:
This is used here https://github.com/crate/crate/blob/732e300660f42c8e2c00c5b1e663f7010ccef016/server/src/test/java/io/crate/planner/operators/JoinTest.java#L109 
and it's needed, because the logic moved into the optimizer.

Reply:
I think it's acceptable like this, since it's package visible and only used for testing

Reply:
yep, ok I missed this, thx!